ci-operator/config/openshift/image-registry/openshift-image-registry-release-4.3.yaml

```diff
@@ -24,7 +24,7 @@ resources:
       memory: 4Gi
     requests:
       cpu: 100m
-      memory: 200Mi
+      memory: 1Gi
```

Comment:
```suggestion
      memory: 200Mi
  'integration':
    limits:
      memory: 8Gi
```

Reply:
I tried to use 6 Gi in https://github.com/openshift/release/pull/5201, but it wasn't enough.